apps/mull-ui/src/app/pages/event-page/event-page-info/event-page-info.scss

```diff
@@ -18,18 +18,14 @@
 }
 
 .cancel-modal-container {
-  position: relative;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  overflow: hidden;
+  border-radius: 1rem !important;
```

Comment:
💡 Just FYI all this copy-paste is probably not a good idea. I know that we're nearing the end of the project, but if someone ends up needing to make another modal we really need to make a component out of it.

Reply:
Yeah , we can create a ticket to create a custom component as we would also need to make sure the modal is good for the identified trash front-end and this should probably be merged to unblock the other tickets tonight